mythtv/programs/mythbackend/encoderlink.cpp

```diff
@@ -330,8 +330,8 @@ bool EncoderLink::CheckFile(ProgramInfo *pginfo)
         ReferenceLocker rlocker(m_sock);
         return m_sock->CheckFile(pginfo);
     }
-
     pginfo->SetPathname(GetPlaybackURL(pginfo));
+
```

Comment:
Please fix up the blank lines. No sense in commiting just a white space change.

Reply:
Done